docs/Node Operators/configuration-variables.md

```diff
@@ -420,6 +435,102 @@ Setting this number higher will cause Chainlink to select higher gas prices.
 
 Setting it lower will tend to set lower gas prices.
 
+# Debugging
+
+## Logging
+
+### JSON_CONSOLE
+
+- Default: `"false"`
+
+Set to true to enable JSON logging. Otherwise will log in a human-friendly console format.
+
+### LOG_LEVEL
+
+- Default: `"info"`
+
+The `LOG_LEVEL` environment variable determines both what is printed on the screen and what is written to the logfile, located at `$ROOT/log.jsonl`.
+
+The available options are:
+- "debug"
+- "info"
+- "warn"
+- "error"
+- "panic"
```

Comment:
```suggestion
The available options are:
- `debug`
- `info`
- `warn`
- `error`
- `panic`
```

Reply:
Could we consider leaving out `panic`, so that we aren't encouraging suppression of errors?